tests/unit/test_report_junit.py

```diff
@@ -18,7 +18,7 @@ def report_fix(tmpdir):
 
     out_file_path = str(tmpdir.join("out.xml"))
 
-    def opt(key, default=None):
+    def get(key, default=None):
         if key == 'verbose':
```

Comment:
I assume that we can also remove the first `if` branch completely. `verbose` is an option, not metadata, `self.get("verbose")` is never used (it's always `self.opt("verbose")`) so mocking the result of it does not make much sense. I don't see verbose being used anywhere in the plugin so we can omit mocking it altogether.